XenAdmin/Actions/GUIActions/DownloadUpdatesXmlAction.cs

```diff
@@ -38,6 +38,7 @@
 using XenAdmin.Core;
 using System.Diagnostics;
 using System.Net;
+using System.Text;
 

```

Comment:
I think we should keep DownloadUpdatesXmlAction in XenModel, and pass in the user agent and user agent id that we want to use, rather than move it all to XenAdmin.